src/CalendarAvailability.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-	<ul class="week-day-container">
+	<ul class="week-day-container" aria-label="List of weekdays">
```

Comment:
```suggestion
	<ul class="week-day-container" aria-label="l10nList">
```
To be translatable we need to add an `l10nList` prop and when passing the actual prop is should be just "Weekdays" as assistive technologies already know that ul is a list

Reply:
or simply `:aria-label="t('Weekdays')"`?

Reply:
That won't work. The transifex integration is not set up for this library. Hence, there are all those `l10n*` props on the main component so that apps can pass their translates strings.

Reply:
ah i see

Reply:
Agreed, I added this change with the next PR. AFAIK, the places that will need these updates for our supported apps are the calendar app and default settings.

Reply:
I made the props default null and not required in case a user does not feel the need to use this on their small (potentially personal) website / app 👍